PyPortal_Azure_Plant_Monitor/azure_gfx_helper.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, is_celsius=True):
         self.temp_label.y = 65
         self._text_group.append(self.temp_label)
 
-        self.temp_text = Label(self.data_font, max_glyphs=10)
+        self.temp_text = Label(self.data_font, max_glyphs=10, text="")
```

Comment:
This is a workaround for https://github.com/adafruit/Adafruit_CircuitPython_Display_Text/issues/45. This can be reverted once the fix is released.